frontend/app/assets/javascripts/spree/frontend/views/spree/layouts/spree_application.js

```diff
@@ -32,4 +32,20 @@ Spree.ready(function ($) {
       }
     }
   };
+
+  var navBarCategoryLinks = document.getElementsByClassName('main-nav-bar-category-links')
+  var navBarCategoryButtons = document.getElementsByClassName('main-nav-bar-category-button')
+  var navBarCategoryImages = document.getElementsByClassName('category-image')
+  var navbarLinks = [navBarCategoryLinks, navBarCategoryButtons, navBarCategoryImages]
+
+  $.each(navbarLinks, function(index, navbarElements) {
+    $.each(navbarElements, function(index, navBarCategoryLink) {
+      navBarCategoryLink.addEventListener('click', function () {
+        console.log(navBarCategoryLink)
```

Comment:
do you need this `console.log` ?

Reply:
It was just for test purposes and I've forgotten to remove it. Will correct it, right away.